app/tests/test_urls.py

```diff
@@ -0,0 +1,9 @@
+from django.test import Client, TestCase
+from django.urls import reverse
+
+
+class test_accounts_page(TestCase):
+    def test_getLogin(self):
+        client = Client()
+        response = client.get("http://127.0.0.1:8000/accounts/login/")
+        self.assertEqual(response.status_code, 200)
```

Comment:
While this is the proper syntax for TestCase, I would prefer ```assert response.status_code == 200``` as this will better match other tests that will be run.

Reply:
Done wait for commit

Reply:
Addressed in 
4a254ad